models/book.rb

```diff
@@ -0,0 +1,22 @@
+require_relative 'rental'
+
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title:, author:)
+    @title = title
+    @author = author
+  end
+
+  def rentals
+    Rental.all.select { |rental| rental.book == self }
+  end
+
+  def people
+    rentals.map(&:person)
+  end
+
+  def add_rental(rental)
+    rental.instance_variable_set(:@book, self)
+  end
```

Comment:
- [ ] In order to create the `has_many` side of `Book` and `Rental`, we need to initialize `@rentals` array in the constructor. This also means that you would need to remove the `rentals` method because it would no longer be necessary 👍 
- [ ] Consequently, your `add_rental` method would be rewritten bearing in mind that the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association
```ruby
def add_rental(person, date)
  Rental.new(date, person, self)
end
```